templates/issues/index.jsx

```diff
@@ -24,11 +24,13 @@ const IssuesTemplate = ({
         lastHarvestingDate={lastHarvestingDate}
       />
       <ErrorsOverviewCard totalCount={totalCount} errorsCount={errorsCount} />
-      <TableCard
-        totalCount={totalCount}
-        aggregation={aggregation}
-        pages={pages}
-      />
+      {pages && (
```

Comment:
Why do you add such conditions everywhere? What is the problem?

Reply:
if you access the page directly the variable `pages` is undefined. This happens when you try to access a repository which is not in your organisation as a superadmin because data provider is not loaded yet.